### PR TITLE
fix(bulletin): read allowance fields from AuthorizationExtent

### PR DESCRIPTION
**Pull request by Imod7**
### Problem
`checkAuthorization` returns wrong values against post-[#448](https://github.com/paritytech/polkadot-bulletin-chain/pull/448) Bulletin chains. After a fresh `authorize_account(N, M)`, `auth.remainingTransactions` returns `0` instead of `N` - and consumers using "if remaining < required, re-authorize" loops re-authorize on every upload.

Not a hard failure (the `authorize_account` extrinsic is additive, so uploads still succeed), but produces unbounded quota growth and an extra extrinsic per upload. Same root cause and shape as the dotns-cli bug recently fixed in [paritytech/dotns-sdk#125](https://github.com/paritytech/dotns-sdk/pull/125) ([@parity/dotns-cli@0.6.1](https://github.com/paritytech/dotns-sdk/releases/tag/v0.6.1)).

### Root cause

After [`paritytech/polkadot-bulletin-chain#448`](https://github.com/paritytech/polkadot-bulletin-chain/pull/448) ("Adds `AllowanceBasedPriority` extension", commit `e543696`, 2026-04-30, picks fields from #469), `AuthorizationExtent` has 5 fields, and the **same field names mean different things**:

  ```rust
  // pallets/transaction-storage/src/lib.rs after #448
  pub struct AuthorizationExtent {
      pub transactions: u32,            // CONSUMED so far  ← was "allowance"
      pub transactions_allowance: u32,  // total ALLOWANCE (new field)
      pub bytes: u64,                   // CONSUMED so far  ← was "allowance"
      pub bytes_permanent: u64,         // (new)
      pub bytes_allowance: u64,         // total ALLOWANCE (new field)
  }
  ```

The SDK's `checkAuthorization` was reading `auth.extent.transactions` / `auth.extent.bytes` and exposing them as `remainingTransactions` / `remainingBytes` - semantics that were correct pre-#448 but inverted after.

The bundled papi metadata snapshot was also pre-#448, so the typed API didn't expose the new fields either.

### Fix

Compute `remaining = allowance − consumed` from the new fields. Public `AuthorizationStatus` contract unchanged. `remainingTransactions: number` and `remainingBytes: bigint` keep working. Consumer code (e.g. mark3t's `packages/bulletin/src/standalone.ts:ensureAuthorized`) needs no change after picking up `^0.2.1`.

- **Source** (`packages/bulletin/src/authorization.ts`): `remainingTransactions: auth.extent.transactions_allowance - auth.extent.transactions`, same
  shape for bytes.
- **Tests** (in-source vitest block): updated 4 existing mocks to the post-#448 5-field struct, added a new "partial-use" test asserting `remaining =   allowance − consumed` for non-boundary values.
- **Metadata**: `papi update bulletin` refreshed `packages/descriptors/.papi/metadata/bulletin.scale` to the post-#448 5-field struct (live `wss://paseo-bulletin-rpc.polkadot.io`, `spec_version: 1,000,013`). Side-effect file changes: `chains/bulletin/.papi/polkadot-api.json` (codeHash bump) and `chains/bulletin/generated/package.json` (autogen artifact).

### Test plan

- [x] `pnpm -F @parity/product-sdk-bulletin build` - clean (ESM + DTS)
- [x] `pnpm -F @parity/product-sdk-bulletin test` - 71/71 passing (14 in `authorization.ts`, including the new partial-use test)
- [x] `pnpm check` - biome green
- [x] Refreshed metadata contains all 5 post-#448 fields (`transactions_allowance`, `bytes_allowance`, `bytes_permanent` verified by grep)

### CODE REVIEW

File: product-sdk/.changeset/auth-extent-allowance-fields.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (TarikGul) [inline product-sdk/.changeset/auth-extent-allowance-fields.md]: We should move the changeset md's to a new root level folder called pending-changesets. That way it doesn't trigger a release. The problem being once this is merged it will trigger a release since this md exists in ./.changesets

**Comment by TarikGul** [comment]
PR looks good, just need to move the changeset md to a different place

File: product-sdk/.changeset/auth-extent-allowance-fields.md (hunk 1 above)
Comment (Imod7) [reply product-sdk/.changeset/auth-extent-allowance-fields.md]: I deleted the file.

**Review by TarikGul — APPROVED** [review]
